sys-apps/mlocate/mlocate-0.26-r3.ebuild

```diff
@@ -45,6 +46,9 @@ src_install() {
 	newins "${FILESDIR}"/mlocate.cron-r3 mlocate
 	fperms 0755 /etc/cron.daily/mlocate
 
+	systemd_dounit "${FILESDIR}"/updatedb.service
+	systemd_dounit "${FILESDIR}"/updatedb.timer
```

Comment:
We call updatedb to upgrade the locate database, but I share your opinion to change the name.